src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SDSEarlyReleaseDefaultingRulesService.kt

```diff
@@ -214,7 +221,7 @@ class SDSEarlyReleaseDefaultingRulesService(
         releaseDate = commencementDate,
         unadjustedDate = early,
       )
-    } else if (standard != null && ((commencementDate != null && standard.isBefore(commencementDate)) || commencementDate == null)) {
+    } else if (standard != null && (commencementDate == null || standard.isBefore(commencementDate) || standard == early)) {
```

Comment:
added an extra condition here + tweaked the logic to make it more concise

new condition `if standard==early` then standard takes precedence and those rules are enforced

The specific rule that relates it to standard  (as discussed with the analyst) is 'The CRD comes from a SDS with an excluded offence so the release date is calculated at 50% not 40%. '. Hiwever at this level we dont have access to that.. therefore gone with `standard == early`